content_resolver/analyzer.py

```
@@ -397,248 +475,398 @@ def print_metrics(self):
+                        if repo_name in error_msg:
+                            if repo_name not in failed_repos:
+                                log(f"  Disabling problematic repository: {repo_name}")
+                                failed_repos.append(repo_name)
+                                try:
+                                    repo_query = RepoQuery(base)
+                                    for repo_weak_ptr in repo_query:
+                                        repo_obj = repo_weak_ptr.get()
+                                        if repo_obj.get_id() == repo_name:
+                                            repo_obj.disable()
+                                            break
+                                except Exception as disable_err:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)